system/lib.rs

```diff
@@ -13,42 +13,46 @@ use std::{fmt::format, sync::Arc};
 use database::{database_manager::DatabaseManager, Database};
 use storage::durability_client::WALClient;
 use typeql;
-
+use database::database_manager::INTERNAL_DATABASE_PREFIX;
 use crate::{repositories::SCHEMA, util::transaction_util::TransactionUtil};
 
-const SYSTEM_DB: &str = "system";
+fn get_system_database_name() -> String {
+    format!("{}system", INTERNAL_DATABASE_PREFIX)
+}
```

Comment:
Why don't you keep this as as `&'static str`? You can use `concat!` macro to create this string statically: https://doc.rust-lang.org/std/macro.concat.html